### PR TITLE
Add job to create a tweet on release

### PR DESCRIPTION
**Pull request by Kludex**
## Change Summary

Add job to create a tweet on release.

## Notes

This is tested locally...

## Checklist for Samuel
- [ ] Set the following environment variables on GitHub:
   - TWITTER_CONSUMER_KEY
   - TWITTER_CONSUMER_SECRET
   - TWITTER_ACCESS_TOKEN
   - TWITTER_ACCESS_TOKEN_SECRET

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -365,3 +369,39 @@ jobs:
...
+          import os
+          import tweepy
+
+          client = tweepy.Client(
+              access_token=os.getenv("TWITTER_ACCESS_TOKEN"),
+              access_token_secret=os.getenv("TWITTER_ACCESS_TOKEN_SECRET"),
+              consumer_key=os.getenv("TWITTER_CONSUMER_KEY"),
+              consumer_secret=os.getenv("TWITTER_CONSUMER_SECRET"),
+          )
+          tweet = base64.b64decode(os.environ["TWEET"].format()).decode("utf-8")
```
Comment (Kludex) [inline .github/workflows/ci.yml:397]:
```suggestion
          tweet = base64.b64decode(os.environ["TWEET"]).decode("utf-8")
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -365,3 +367,39 @@ jobs:
...
+          import os
+          import tweepy
+
+          client = tweepy.Client(
+              access_token=os.getenv("TWITTER_ACCESS_TOKEN"),
+              access_token_secret=os.getenv("TWITTER_ACCESS_TOKEN_SECRET"),
+              consumer_key=os.getenv("TWITTER_CONSUMER_KEY"),
+              consumer_secret=os.getenv("TWITTER_CONSUMER_SECRET"),
+          )
+          tweet = base64.b64decode(os.environ["TWEET"]).decode("utf-8")
```
Comment (Kludex) [inline .github/workflows/ci.yml:395]:
```suggestion
          tweet = os.getenv("TWEET")
```

**Comment by Kludex** [comment]
@samuelcolvin do you want me to do something else here?

**Review by samuelcolvin — APPROVED** [review]
please update to fix conflicts, then this can be merged.

**Comment by samuelcolvin** [comment]
I've set the secrets, so this should be merged as soon as it passes, not sure what's going on with CI.

**Pull request merged**